src/framework/uicomponents/qml/MuseScore/UiComponents/StyledToolTip.qml

```diff
@@ -28,17 +28,24 @@ import MuseScore.UiComponents 1.0
 StyledPopupView {
     id: root
 
+    enum ToolTipType {
```

Comment:
Now the enum is defined both in QML and in C++. (That's partially my bad, because I didn't know you would also need it in C++ so therefore I suggested creating it in QML).

It would be better to use just the C++ definition, which can be used in QML by writing `QmlToolTip.Default` and `QmlToolTip.FileToolTip`.

(The type in `property int toolTipType:` should indeed still be `int` when doing that.)

Reply:
I see, didn't know that was possible.